docs/docs/reference/shards/lisp/misc.md

```diff
@@ -33,44 +33,92 @@ This is useful for cases where you need multiple shards (to transform your data)
 
 ## apply
 
-```clojure linenums="1"
+```
+(apply f args) (apply f x args) (apply f x y args) (apply f x y z args)(apply f a b c d & args)
+```
+
+Applies fn f to the argument list formed by prepending intervening arguments to args.
 
+```clojure linenums="1"
+(apply str ["cherry " "banana " "apple"])
+;;=> "cherry banana apple"
 ```
 
 ## atom
 
-```clojure linenums="1"
+```
+(atom x)
+```
 
+  Creates and returns an Atom with an initial value of x and zero or more options (in any order).
+
+```clojure linenums="1"
+(atom {})
 ```
 
 ## deref
 
-```clojure linenums="1"
+```
+(deref ref)
+```
+
+Returns the most-recently-committed value of ref. When applied to an atom, returns its current state.
```

Comment:
What does `most-recently-committed` mean here? can you link it to an explanation or add a short one below.

Reply:
I'll remove such troublesome explanation.